rgw/v2/lib/admin.py

```diff
@@ -38,8 +38,18 @@ def create_admin_user(self, user_id, displayname, cluster_name='ceph'):
             write_user_info = AddUserInfo()
             basic_io_structure = BasicIOInfoStructure()
             log.info('cluster name: %s' % cluster_name)
-            cmd = 'radosgw-admin user create --uid=%s --display-name=%s --cluster %s' % (
-                user_id, displayname, cluster_name)
+            log.info('Verify if the cluster is a secondary or primary site via radosgw-admin sync status')
+            sync_status_cmd = 'radosgw-admin sync status'
+            pr = subprocess.Popen(sync_status_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True,
+                                  shell=True)
+            out, err = pr.communicate()
+            primary_site = out.find('zone is master')
```

Comment:
this can definitely be added outside of this function. see if this can be added in utils, so that we can reuse this !